docs/readthedocs/source/doc/Orca/Overview/known_issues.md

```diff
@@ -38,6 +38,34 @@ If you are using `init_orca_context(cluster_mode="yarn-client")`:
    spark-submit --conf spark.python.worker.reuse=false
    ```
 
+### **RuntimeError: Inter op parallelism cannot be modified after initialization**
+
+This error occurs while building model in driver rather than in worker. You should build the model in model creator which runs in each ray worker. You can refer to the following example:
```

Comment:
This error occurs if you build your TensorFlow model on the driver rather than in worker. You should build the complete model in `model_creator` which runs on each worker node. You can refer to the following examples:

Reply:
fixed.